ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/controlModules/foot/toeOff/DynamicStateInspector.java

```diff
@@ -172,11 +173,11 @@ public void checkICPLocations(DynamicStateInspectorParameters parameters,
       {
          this.isDesiredECMPOKForToeOff.set(true);
       }
-      if (currentCoP != null)
+      if (currentCoP != null && Double.isFinite(percentLoad) && percentLoad > parameters.getLoadThresholdToCheckCoP())
```

Comment:
If the foot isn't sufficiently loaded, it doesn't matter where the CoP is, because it won't make much of a change to the system.